include/TrackView.h

```diff
@@ -98,6 +98,12 @@ class TrackView : public QWidget, public ModelView, public JournallingObject
 		return m_action == Action::Move;
 	}
 
+	// returns getTrackFixedSettingsWidth() + getTrackFixedOperationsWidth()
+	static const int getTrackFixedWidth();
+	// returns the right compact or not compact width
+	static const int getTrackFixedSettingsWidth();
+	static const int getTrackFixedOperationsWidth();
```

Comment:
You can make this documentation better.

Also, I removed the "Fixed" part since I don't think callers of these functions care about if its a fixed width or not. It is simply the width of the thing. If you go through with this change, you'll also have to of course update the function declarations and definitions to reflect the new name.

```suggestion
	//! Returns the width of a track.
	static const int getTrackWidth();

	//! Returns the width of the settings portion of a track, based on whether or not the track is compacted.
	static const int getTrackSettingsWidth();
	
	//! Returns the fixed width of the operations portion of a track, based on whether or not the track is compacted.
	static const int getTrackOperationsWidth();
```